classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../item'
+
+class Book < Item
```

Comment:
Optional.
As per the UML diagram, a Book belongs to the Item class, and every item should possess an author, label, and genre. However, in your case Book only asks for the label. Since previous reviewers didn't raise this as required, I am marking this as optional.